components/motor/gpio/motor_encoder.go

```diff
@@ -428,17 +440,22 @@ func (m *EncodedMotor) computeNewPowerPct(currentRPM, desiredRPM float64) float6
 
 func (m *EncodedMotor) rpmMonitorPassSetRpmInLock(currentRPM, desiredRPM, rotationsLeft float64, rpmDebug bool) {
 	lastPowerPct := m.state.lastPowerPct
-	m.state.currentRPM = currentRPM
 
 	newPowerPct := m.computeNewPowerPct(currentRPM, desiredRPM)
-	if newPowerPct == lastPowerPct {
-		return // No changes to power are needed right now
+	if newPowerPct == lastPowerPct { // No changes to power are needed right now
+		if rpmDebug {
+			m.logger.Debugf("newPowerPct %.2f equals lastPowerPct %.2f", newPowerPct, lastPowerPct)
+		}
+		return
 	}
 
 	if rpmDebug {
 		m.logger.Debugf("current rpm: %0.1f desiredRPM: %0.1f power: %0.1f -> %0.1f rot2go: %0.1f",
 			currentRPM, desiredRPM, lastPowerPct*100, newPowerPct*100, rotationsLeft)
+		m.logger.Debugf("setting newPowerPct to %.2f", newPowerPct)
```

Comment:
Why make a second call to `m.logger.Debugf` instead of rolling it into the previous line? I don't feel strongly either way; it just seems a little unusual.

Reply:
my new line preference/laziness.